src/core/PlayAreaApi.ttslua

```diff
@@ -83,5 +84,30 @@ do
     return getObjectFromGUID(PLAY_AREA_GUID).call("isInPlayArea", object)
   end
 
+  -- Called by Custom Data Helpers to push their location data into the Data Helper.  This adds the
+  -- data to the local token manager instance.
+  ---@param args Table Single-value array holding the GUID of the Custom Data Helper making the call
+  PlayAreaApi.updateLocations = function(args)
+    getObjectFromGUID(PLAY_AREA_GUID).call("updateLocations", args)
+  end
+
+  PlayAreaApi.getCustomDataHelper = function()
+    return getObjectFromGUID(PLAY_AREA_GUID).getVar("customDataHelper")
+  end
+
+  PlayAreaApi.getCustomObject = function()
+    return getObjectFromGUID(PLAY_AREA_GUID).getCustomObject()
+  end
+
+  PlayAreaApi.setCustomObject = function(customInfo)
+    return getObjectFromGUID(PLAY_AREA_GUID).setCustomObject(customInfo)
+  end
+
+  PlayAreaApi.reload = function()
+    return getObjectFromGUID(PLAY_AREA_GUID).reload()
+  end
```

Comment:
Why do we need these as functions in the API?

Reply:
Usually we add an Api if we need to access resources (variables or functions) of another object. So for this case I'd say it would be better if you just add a new function that takes the new image as parameter instead of 3 functions that are useless on their own.